### DISCUSSION TITLE
Why is the voldemort notebook downloading ckpt file when I already have it in my gdrive?

### DISCUSSION THREAD

**Discussion by G-force78** [hlky/stable-diffusion-webui#1373] [Q&A]
The title sums it up. 
I already have the v1.4 ckpt file in my gdrive folder yet the collab is now downloading it again to  /content/stable-diffusion-webui/models/Stable-diffusion/sd-v1-4.ckpt

Looks like google have strangled download speed for free use accounts too.

**Comment by hlky** [comment]
Perhaps you should check with the contributors of the repo you are using.
